components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -112,31 +188,146 @@ function NavigationListItem(item: NavItem, props: DrawerNavGroupProps): ReactNod
         }
     };
 
+    const chevron = groupChevron === undefined ? itemChevron : groupChevron;
+
+    function getRightActionIcon(): JSX.Element {
+        if (item.rightComponent) {
+            return item.rightComponent;
+        }
+        if (chevron) {
```

Comment:
If I pass in chevron:true, the list item is not able to be expanded/collapsed

Reply:
Should we be joining what they pass in as rightComponent with our expand icon? So they can still put whatever they want and we can easily control the expand/collapse behavior?

Reply:
> If I pass in chevron:true, the list item is not able to be expanded/collapsed

Not true. They are expandable. The icon and right component action thing are separate ...

Reply:
Now if I pass rightComponent and chevron:true, there's no chevron and no expand/collapse icon

Reply:
chevron is a shortcut for rightComponent...it shouldn't be included in the logic for expand/collapse icon

Reply:
So now if I have rightComponent, chevron and expandIcon all be defined, it will look like this?
![image](https://user-images.githubusercontent.com/8997218/75720697-f3a09200-5ca4-11ea-9a76-3b0196c34a78.png)